features/facebookImport/src/facebookImporter.jsx

```diff
@@ -37,18 +35,18 @@ const FacebookImporter = () => {
 
     const { files } = useContext(PolyImportContext);
 
+    const navigate = useNavigate();
+
     function determineRoute() {
         if (files.length > 0)
-            return (
-                <Redirect
-                    to={{
-                        pathname: "/overview",
-                        state: INITIAL_HISTORY_STATE,
-                    }}
-                />
-            );
-        else return <Redirect to={{ pathname: "/import" }} />;
+            return navigate("/overview", { state: INITIAL_HISTORY_STATE });
```

Comment:
Gotta double check if the return value of `determineRoute` is used for anything - because `navigate` has no return value.